openmc/stats/multivariate.py

```diff
@@ -380,7 +380,7 @@ class SphericalIndependent(Spatial):
     """Spatial distribution represented in spherical coordinates.
 
     This distribution allows one to specify coordinates whose :math:`r`,
-    :math:`\theta`, and :math:`\phi` components are sampled independently from
+    :math:`\\theta`, and :math:`\phi` components are sampled independently from
```

Comment:
Ah, the proper thing to do here would be to make the docstring be a raw string by adding an `r` prefix:
```Python
r"""Raw string where \t doesn't get interpreted as a tab character"""
```